quote-generator/src/App.js

```
@@ -0,0 +1,30 @@
+import {useState, useEffect} from "react";
+import './App.css';
+
+function App() {
+  const [quotes, setQuotes] = useState([]);
+
+  useEffect(() => {
+    fetch('https://quotes-server-by-tony.glitch.me/quotes').then((response) =>{
+      return response.json();
+    })
+    .then((data)=> {
+      setQuotes(data);
+    });
```

Don't forget to add a `.catch` clause to your promises, otherwise if the promise throws an error it will get swallowed, you won't know about it. eg as a minimum:
`.catch(err => console.error(err))`